lib/associations/base.js

```diff
@@ -23,7 +23,7 @@ const AssociationError = require('./../errors').AssociationError;
  * user.getPictures() // gets you all pictures
  * user.getProfilePicture() // gets you only the profile picture
  *
- * User.findAll({
+ * User.findOneAll({
```

Comment:
Is that intended ?

Reply:
No that must be a regex gone haywire, no bueno, will fix once @sushantdhiman decided which of my changes are applicable.

Reply:
Please don't wait up for me, I don't think I will have time to review 80+ files. As I suggested earlier it will be better to propose list of duplicate / removable targets, then we can discuss which one we need to actually deprecate / remove